src/UI/Mail/Keybindings.hs

```diff
@@ -55,7 +55,14 @@ displayMailKeybindings =
                                              `chain` continue)
     , Keybinding (V.EvKey (V.KChar '?') []) (noop `focus` continue @'Help @'ScrollingHelpView)
     , Keybinding (V.EvKey (V.KChar 'r') []) (
-        replyMail
+        senderReply
+        `focus` (
+            invokeEditor ViewMail ScrollingMailView
+            :: Action 'ComposeView 'ComposeListOfAttachments (T.Next AppState)
+            )
+        )
+    , Keybinding (V.EvKey (V.KChar 'R') []) (
```

Comment:
Wasn't it `g` in mutt? Either way not a big deal ...

Reply:
dunno... I have not touched my mutt keybindings in 15 years, but I see I have explicitly bound it to `R`.  If the mutt default is 'g' I am happy to change it.

Reply:
You're correct; the default is 'g': http://www.mutt.org/doc/manual/  .  I will change it.